### PR TITLE
Side 1244 Add links to list whispers

### PR DESCRIPTION
**Pull request by BrettMugglinOlive**

### CODE REVIEW

**Review by dmorrison-olive — APPROVED** [review]

File: ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/List.cs [hunk 1]
```diff
@@ -23,22 +23,30 @@ public enum Align
...
 
     public class ListMessage : ListBase
     {
-        public string Header { get; }
-        public string Body { get; }
-        public Style Style { get; }
-        public Align Align { get; }
+        public string Header { get; set; }
+        public string Body { get; set; }
+        public Style Style { get; set; }
```
Comment (bbrewer-oliveai) [inline ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/List.cs:36]: As we discussed, these never should have worked, as you can't really mutate a string through a getter like you could a List or Dictionary

File: ldk/node/src/hostClients/whisperMessageBuilder.ts [hunk 2]
```diff
@@ -230,6 +230,27 @@ export const generateWhisperListElement = (
       msg.setDivider(inputMsg);
       break;
     }
+    case 'link': {
+      const inputMsg = new WLE.Link();
+      if (element.align) {
+        inputMsg.setAlign(generateWhisperListAlign(element.align));
+      } else {
+        inputMsg.setAlign(generateWhisperListAlign(WhisperListAlign.LEFT));
```
Comment (richseviora) [inline ldk/node/src/hostClients/whisperMessageBuilder.ts:238]: Does it make sense consolidate this into the line above?

something like

```typescript
inputMsg.setAlign(generateWhisperListAlign(element.align || WhisperListAlign.LEFT));
```

**Review by richseviora — APPROVED** [review]
Optional change - otherwise looks great!

**Review by bbrewer-oliveai — APPROVED** [review]

File: ldk/node/src/hostClients/whisperMessageBuilder.ts (hunk 2 above)
Comment (BrettMugglinOlive) [reply ldk/node/src/hostClients/whisperMessageBuilder.ts:238]: Made that change

**Pull request merged**